docs/File-Formats.md

```diff
@@ -677,6 +677,18 @@ The `cbp_driver` column flags the mutation as either driver or passenger. In cBi
 You can learn more about configuring these annotations in the [portal.properties documentation](portal.properties-Reference.md#custom-annotation). When properly configured, the customized annotations appear in the "Mutation Color" menu of the OncoPrint view: \
 ![schreenshot mutation color menu](images/screenshot-mutation-color-menu.png) 
 
+#### Allele specific copy number (ASCN) annotations
+Allele specific copy number (ASCN) annotation is also supported and may be added using namespaces, described [here](#adding-mutation-annotation-columns-through-namespaces). If ASCN data is present in the MAF, the deployed cBioPortal instance will display additional columns in the mutation table showing ASCN data. All of the following columns must be present for ASCN data to show up in portal.
+
+42. **ASCN.ASCN_METHOD (Optional)**: "FACETS" or any method used to obtain ASCN data
+43. **ASCN.CCF_M_COPIES (Optional)**: 
+44. **ASCN.CCF_M_COPIES_UPPER (Optional)**:
+45. **ASCN.MUTANT_COPIES (Optional)**:
+46. **ASCN.CLONAL (Optional)**: "True" or "False"
+47. **ASCN.TOTAL_COPY_NUMBER (Optional)**: total copy number of the gene
+48. **ASCN.MINOR_COPY_NUMBER (Optional)**: copy number of the minor allele
+49. **ASCN.ASCN_INTEGER_COPY_NUMER (Optional)**: 
```

Comment:
If someone wishes to support ASCN annotations in their MAF then are these fields *required* or optional for ASCN support? A distinction should be made here if these fields are actually required for ASCN support.